armi/bookkeeping/db/databaseInterface.py

```diff
@@ -151,36 +151,24 @@ def interactEveryNode(self, cycle, node):
         if self.o.cs["tightCoupling"]:
             # h5 cant handle overwriting so we skip here and write once the tight coupling loop has completed
             return
-        self.writeDBEveryNode(cycle, node)
+        self.writeDBEveryNode()
 
-    def writeDBEveryNode(self, cycle, node):
+    def writeDBEveryNode(self):
         """Write the database at the end of the time node."""
-        # skip writing for last burn step since it will be written at interact EOC
-        if node < self.o.burnSteps[cycle]:
-            self.r.core.p.minutesSinceStart = (
-                time.time() - self.r.core.timeOfStart
-            ) / 60.0
-            self._db.writeToDB(self.r)
-            if self.cs[CONF_SYNC_AFTER_WRITE]:
-                self._db.syncToSharedFolder()
+        self._db.writeToDB(self.r)
+        if self.cs[CONF_SYNC_AFTER_WRITE]:
+            self._db.syncToSharedFolder()
 
     def interactEOC(self, cycle=None):
-        """In case anything changed since last cycle (e.g. rxSwing), update DB. (End of Cycle)."""
-        # We cannot presume whether we are at EOL based on cycle and cs["nCycles"],
-        # since cs["nCycles"] is not a difinitive indicator of EOL; ultimately the
-        # Operator has the final say.
-        if not self.o.atEOL:
-            self.r.core.p.minutesSinceStart = (
-                time.time() - self.r.core.timeOfStart
-            ) / 60.0
```

Comment:
Do we not want to update the time any more?

Reply:
we tend to only do it before db write since its not used anywhere in code.. not that it is stored in EOL anyway

Reply:
Sorry, I still don't understand.

> we tend to only do it before db write since its not used anywhere in code.

So, did we add this somewhere else?

Reply:
Mark's these is that we only need to update this parameter when we are writing to the DB, which is no longer happening here.

I can confirm that (in ARMI and the broader ecosystem), people are only using the parameter in reference to a DB.  If that helps.

Reply:
yes i talked with john to explain. its only a valuable calculation when you are about to write a db. so only do the work then. its calculated at EOC and before all other writes 


ooooooHHHH, yes we still want it on the every node. not sure how that got lost.

Reply:
its back now